proof-of-concept/setup.py

```diff
@@ -14,6 +14,7 @@
 
 setup(
     name="hpy-pof",
+    packages = ['pofpackage'],
```

Comment:
Are you sure this is correct? The import is failing

Reply:
No, I'm not *sure* but I think it is required. As I wrote in the IRC channel: when I manually activate the venv created by `test_pof.sh`, running `import pofpackage.foo` also fails without that. Specifying `packages = ["pofpackage"]` fixed this. But the import still fails when running the tests.